storage/aptosdb/src/lib.rs

```diff
@@ -1294,8 +1294,7 @@ impl DbWriter for AptosDB {
         })
     }
 
-    /// Snapshots are persisted checkpoints that merklize global state key-value pairs.
-    fn save_state_snapshot(
+    fn save_state_snapshot_for_bench(
```

Comment:
What's the plan for this function next?

Reply:
After I finishes lazy db, it will be deleted and exec-bench will also delete the usage of this function since committing is handled by db.